projects/canopy/src/lib/filter-buttons/filter-button/filter-button-select-multiple.stories.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { FormBuilder, FormGroup, FormArray } from '@angular/forms';
+
+import { FilterVariant } from '../filter-button.interface';
+
+@Component({
+  selector: 'lg-reactive-form-multiple',
+  template: `
+    <form [formGroup]="form">
+      <lg-filter-group
+        [variant]="variant"
+        [valueArrayName]="valueArrayName"
+        [formArrayName]="valueArrayName"
+      >
+        {{ label }}
+        <lg-filter-button value="red">Red</lg-filter-button>
+        <lg-filter-button value="yellow">Yellow </lg-filter-button>
```

Comment:
minor again, but some spaces around the text (and on Blue)...
```suggestion
        <lg-filter-button value="yellow">Yellow</lg-filter-button>
```

Reply:
Done.